src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -45,34 +46,32 @@ export class ArtworkSidebarPartnerInfo extends React.Component<
 
   render() {
     const { artwork } = this.props
-    const locationCities = artwork.partner.locations.map((location, index) => {
-      return location.city
-    })
-    const filteredForDuplicatesAndBlanks = locationCities.filter(
-      (city, pos) => {
-        return city && locationCities.indexOf(city) === pos && city.length > 0
-      }
-    )
-
+    const locationNames =
+      artwork &&
+      artwork.partner &&
+      artwork.partner.locations &&
+      artwork.partner.locations.length > 0 &&
```

Comment:
We need to figure out how to add in some kind of `foo.?bar.?` existential operator...

Reply:
Yea, started doing this after yesterday's crashes. I think it's a good practice in general...

Reply:
Definitely -- so many crashes around not guarding things!